kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/test/java/org/kie/server/services/jbpm/ui/form/render/FormRendererTest.java

```diff
@@ -242,12 +245,16 @@ public void testRenderOfBasicTaskFormWithSelectRadioGroupAndData() {
         inputs.put("radio", "radio2");
         inputs.put("decimal", 123.5);
         inputs.put("hwSpec_", "name####123####111####id");
+        inputs.put("startDate_", LocalDateTime.of(LocalDate.of(2020, 12, 1), LocalTime.now()));
         Map<String, Object> outputs = new HashMap<>();
         
+        
         Task task = newTask(0L, "Ready");
         
         String renderedForm = renderer.renderTask("", task, form, inputs, outputs);
         assertThat(renderedForm).isNotNull();
+        assertThat(renderedForm).contains("value=\"2020-12-01");
```

Comment:
I would suggest to check for whole timestamp format.

Reply:
The point of the JIRA is to print just the date part of a LocalDateTime

Reply:
The point of this Jira is to fix handling of LocalDateTime, I'd expect we also cover cases including time part. I don't dispute that the case of date part only is fixed, my concern is if this solution is general enough, trying to be proactive here, not just fix what's exactly reported.

Reply:
The point is that form generator should not print the time part or the browser wont render the date.

Reply:
Then we're fixing a different issue than we have, or?
We can have all 3 following data types in DataObject: LocalDateTime, LocalDate, LocalTime. When someone defines LocalDateTime - won't the proposed solution ignore time part of the LocalDateTime also for them?

Reply:
Anyway, I updated the test so it better reflects the scenario that is failing (a POJO being unmarshalled). I hope the intention is clearer now.

Reply:
I did not get the last comment. 
The issue reported was that when using a LocalDateTime withitn a POJO, when the HTML returned by the form REST API is rendered in a web browser, the date does not appear. 
There were two problems. First, Java 8 clases were not properly unmarshall by formReader.extractParameters. This is solver by registering the JavaModule in the ObjectMapper. Second, LocalDateTime default format prints the time part. Then the brower is not able to render me (it can be argued this is a browser issue, but we cannot fix it), so, for LocalDateTime only, we need to properly setup the deserializer with the right format.

Reply:
I changed the approach so time part is not longed ignored in rendering, by changing HTML type and use date or datelocal-time (both are supported by Firefox and Chrome)